test/components/link-community/link-community.vdiff.js

```diff
@@ -0,0 +1,37 @@
+
+import '../../../src/components/community/link-community.js';
+import '../../../src/components/community/button-community.js';
+import { expect, fixture, html } from '@brightspace-ui/testing';
+
+const opts = { viewport: { height: 450 } };
+
+const langArticleMap = {
+	'en': 4545,
+	'ar-SA': 10489,
+	'es-MX': 9214,
+	'pt-BR': 9978,
+	'nl-NL': 7833,
+	'fr-CA': 6739
+};
+
+describe('link-community', () => {
+	it('Should render a link in link mode', async() => {
```

Comment:
Since these names end up in the filename of the diff file, I'd suggest keeping it simple:

```suggestion
	it('link', async() => {
```